IdentityCore/src/cache/MSIDCacheAccessor.h

```diff
@@ -77,6 +78,13 @@
                                          context:(id<MSIDRequestContext>)context
                                            error:(NSError **)error;
 
+- (MSIDRefreshToken *)getRefreshableTokenWithAccount:(MSIDAccountIdentifier *)account
```

Comment:
I'd prefer to add another explicit method saying getPrimaryRefreshToken. Otherwise there might be ambiguity when credentialType passed is e.g. access token.

Reply:
perhaps PRT? perhaps going forward one day, we should simply refer RefreshToken = RT

Reply:
fixed, I call it `getPrimaryRefreshTokenWithAccount` for now, we can revisit it later to make sure it is consistent regarding `primaryRefreshToken`, `primaryRT` and `PRT` in the project :)